### ISSUE TITLE
A question about dependency

### ISSUE THREAD

**Issue by biaocy** [nathanreyes/v-calendar#591]
Hi, I'm just trying out some date-picker components today. I'm search around, and found this beautiful vue component.  @nathanreyes Thank you for developed this beautiful component.

I have a question about dependency issue while I'm installed this component through `npm`.
I saw some warning from install log
```bash
npm WARN deprecated popper.js@1.16.1: You can find the new Popper v2 at @popperjs/core, this package is dedicated to the legacy v1
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.11 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.11: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

+ v-calendar@1.0.0-beta.23
added 2 packages from 3 contributors in 28.733s

49 packages are looking for funding
  run `npm fund` for details
```
 The deprecated warning above isn't the key matter. At first glance, I'm wonder why `popper.js` is needed, then I recall I saw some popover features on the doc, then I checked [package.json](https://github.com/nathanreyes/v-calendar/blob/68cf760978a9604022c46638dd094d9ffac327e1/package.json#L45)'s dependecies property, this component is depend on `popper,js`. So the log make sense.

**TL;DR:
My question is, are you sure about this component is `dependency-free`? Is there any package building knowledge I'm missing?**

**Comment by tafelnl** [comment]
As far as my knowledge reaches, it is untrue that this is dependency-free. I think it is just a legacy remark from when it used to be dependency-free in the past.
